### DISCUSSION TITLE
(Alerte) Text to speech alerte

### DISCUSSION THREAD

**Discussion by ShalltearFR** [skarab42/marv#78] [Ideas]
Avoir la possibilité d'activer ou non via une coche le "text to speech" pour les messages de sub ou bits.
Bien entendu il faudra aussi configurer la langue du narrateur.

**Comment by Arubinu** [comment]
Coucou, ca n'a aucun lien avec Marv, mais si ca t'intéresse, j'ai fait un soft qui lit le tchat Twitch et/ou les Sub, etc ...

https://github.com/Arubinu/AruChat
Tu peux tester la version web en trouvant le lien dans le README, juste la version Web bêta est la version Release téléchargeable (pas fait le changement au niveau web).
Si tu as besoin d'explications, tu peux me trouver mon serveur Discord via ma chaîne Twitch.
